reference-implementation/lib/readable-stream.js

```diff
@@ -700,8 +700,52 @@ class ReadableStreamDefaultReader {
 
     ReadableStreamReaderGenericRelease(this);
   }
+
+  iterator({ cancel = true } = {}) {
+    const O = this;
```

Comment:
To follow the spec more exactly, don't do the `O` aliasing. Here and everywhere below.